supabase/migrations/20231014104130_make_user_id_in_user_profiles_table_unique.sql

```diff
@@ -0,0 +1 @@
+alter table if exists user_profiles add constraint unique_user_id unique (user_id);
```

Comment:
@PCalape Let's just drop the `id` column and use `user_id` as the primary key.

Reply:
Done applying changes @arnellebalane

Reply:
@PCalape You can now delete this migration file, this field is now a primary key in the next migration so no need to manually make it unique.

Reply:
Nice catch @arnellebalane will apply the change

Reply:
Done applying the changes @arnellebalane